docs/tutorials/getting-started-with-csharp/console-teleprompter.md

```diff
@@ -193,7 +193,14 @@ to add that using statement at the top of file:
 using System.Threading.Tasks;
 ```
 
-Run the sample, and check the output.Now, each single word is printed,
```

Comment:
Line 48: Open that file you your favorite text editor. -> Open that file with your favorite text editor.